Felienne/hedy
# Automatically re-directs teachers to for-teachers page on login

**Description**
In this PR we implement a small feature where teachers are re-direct to the for-teachers page instead of the programs page when logging in. For regular users nothing changes. We add an extra boolean called `teacher` to the login response from the back-end. Using this value in the `afterLogin()` function on the front-end to verify where we want to re-direct to.

**Fixes #1899**

**How to test**
Login with both a teacher an a regular account. Notice that in both cases you will be re-directed to a different page. Verify that the re-direct after signup is also still working correctly. A signup re-direct is always set with `teacher = False` as you can't be a teacher account from the start.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
